lib/elixir/lib/macro.ex

```diff
@@ -370,6 +370,30 @@ defmodule Macro do
       iex> Macro.escape({:unquote, [], [1]}, unquote: true)
       1
 
+  ## Comparison to `Kernel.quote/2`
+
+  The `escape/2` function is sometimes confused with `Kernel.SpecialForms.quote/2`,
+  because the above examples behave the same with both. The key difference is
+  best illustrated when the value to escape is stored in a variable.
+
+      iex> Macro.escape({:a, :b, :c})
+      {:{}, [], [:a, :b, :c]}
+
+      iex> quote do: {:a, :b, :c}
+      {:{}, [], [:a, :b, :c]}
```

Comment:
I would put this in the same block above. Ie. Removing the emoty line

Reply:
*empty line

Reply:
Like this?

```
iex> Macro.escape({:a, :b, :c})
{:{}, [], [:a, :b, :c]}
iex> quote do: {:a, :b, :c}
{:{}, [], [:a, :b, :c]}
```